doc/doxygen/src/Configuration.dox

```diff
@@ -0,0 +1,174 @@
+/*!
+ * @page runtime_configuration Configuring OpenAssetIO
+ *
+ * This page covers how to configure OpenAssetIO for a specific
+ * end-user working environment.
+ *
+ * @section runtime_configuration_overview Overview
+ *
+ * @startuml
+ * node Host {
+ *   package OpenAssetIO {
+ *     collections ManagerPlugin
+ *   }
+ * }
+ *
+ * cloud "Asset Management System" as ams
+ *
+ * ManagerPlugin -right-> ams
+ * ams -left-> ManagerPlugin
+ * @enduml
+ *
+ * An OpenAssetIO enabled tool or application is known as a @ref host "Host"
+ * (as it "hosts" the API, tenuous - but naming things is hard). Through
+ * the use of a plugin system, OpenAssetIO allows the host to
```

Comment:
[Consider]
```suggestion
 * (as it "hosts" the API, tenuous - but naming things is hard). Through
 * the use of a manager plugin, OpenAssetIO allows the host to
```
Unsure on this one. My thinking is that we use the term `manager plugin` below without really defining it, and that the "plugin system", isn't really what lets us communicate. The manager plugins let us communicate, and the plugin system lets us load the manager plug ins.

Reply:
Good idea, updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/860/commits/da4b28697cbc15b230dc72e29c13af56e80bdb50).